docs/architecture/decisions/0002-convert-to-static.md

```diff
@@ -0,0 +1,104 @@
+# 2. Convert to Static Website
+
+Date: 2019-01-24
+
+## Status
+
+Accepted
+
+## Context
+
+This documents outlines the decision to convert careers.mozilla.org from a
+Django Website to a Static Website.
+
+
+### Current Architecture
+
+ - Careers is a Django based website hosted on two k8s clusters
+ - Fetches job postings from [Greenhouse](https://www.greenhouse.io/) by polling
+   it's API every 10 minutes.
+ - Uses two MariaDB instances in a master / replica auto-replicating setup to
+   store fetched job postings.
+
+### Why convert to static?
+
+ - Reduce hosting complexity down from two k8s hosted Apps with DB back-end
+   to a couple of S3 buckets and a CDN (or similar)
+ - Reduce hosting costs (k8s clusters, ELBs, databases)
+ - Reduce code maintenance cost (urgent python upgrades / security
+   vulnerabilities) and overall engineering time.
+
+#### What enables us to consider this option?
+
+ - Historically Careers used to have a Admin interface to submit University
+   Days. This has been replaced by a static file named `university_events.yml`
```

Comment:
is `university_events.yml` ever updated? If so, how?

Reply:
Well now that you mention it, it has never been used and it will not work without modifications with the static-fication of the site. So I'll go ahead and remove this functionality altogether

Reply:
https://github.com/mozmeao/lumbergh/pull/373